src/App.css

```diff
@@ -1,38 +1,250 @@
+}
+
+h1, h3, h4 {
+  font-family: 'Archivo Black', sans-serif;
+  margin: 0;
+}
+
+h2, button, p, li {
+  font-family: 'Quattrocento', serif;
+  margin: 0.5rem 0;
+}
+
+h1{
+  font-size: 3rem;
+  margin-bottom: 2rem;
+  text-shadow: 3px 4px 6px #0a0a0a;
+  text-transform: uppercase;
+}
+
+h2{
+  text-shadow: 1px -1px 0 #290900;
+}
+
+form {
+  width: 50rem;
+  padding: 0;
+  margin-top: 2rem;
+}
+
+input {
+  width: 25rem;
+  height: 3.5rem;
+  margin: 0 auto;
+  color: #333;
+  background: rgba(255,255,255,0.4);
+  box-shadow: 0 2px 10px 2px rgba(0,0,0,0.5);
+  border-radius: 5px;
+  border: none;
+  outline: none;
+}
+
+button {
+  color: #0a0a0a;
+  width: 10rem;
+  height: 3.4rem;
+  margin: 0 1rem;
+  background: rgba(255, 255, 255,0.75);
+  box-shadow: 0 2px 10px 2px rgba(0,0,0,0.5);
+  border:solid 1px white;
+  border-radius: 5px;
+}
+
+input:hover, input:focus, button:hover, button:focus {
+  background: rgba(255, 255, 255,0.8);
+  box-shadow: 0 6px 10px 6px rgba(0,0,0,0.8);
+} /* HEADER ENDS HERE */
+
+
+/* MAIN STARTS HERE */
+main {
+  display: flex;
+  justify-content: center;
+  align-items: space-around;
+}
+
+.drinkContainer {
+  background: #FFFFFF;
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+  border-radius: 0.3rem;
+  width: 90%;
+  margin: 2rem;
+  display: flex;
+  justify-content: space-evenly;  
   align-items: center;
+}
+
+.imgContainer {
+  width: 37.5%;
+  height: 90%;
+  margin: 1.5rem 1rem;
+  border-radius: 0.5rem;
+  display: flex;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-items: center;
+}
+
+img {
+  width: 100%;
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+}
+
+.recipeContainer {
+width: 37.5%;
+margin: 1rem 1.5rem;
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: space-evenly;
+color: black;
+margin: 1.25rem;
+}
+
+ul {
+  padding-left: 1rem;
+}
+
+li, p {
+  font-size: 1rem;
+}
+/* MAIN ENDS HERE  */
+
+/* FOOTER STARTS */
+footer{
+  font-size: 16px;
+  text-align: center;
+  padding: 1.5rem 0;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+a:hover, a:focus {
+  color: #333;
+  text-shadow: 1px 1px 1px #0a0a0a;
 }
+/* FOOTER CIRCUMCISED */
+
 
-.App-link {
-  color: #61dafb;
+/* MEDIA QUERIES */
+@media (max-width: 1050px) {
+  h3 {
+    font-size: 1.2rem;
+    letter-spacing: 0.1rem;
+  }
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+@media (max-width: 950px) {
+
+  .drinkContainer {
+    width: 100%;
   }
-  to {
-    transform: rotate(360deg);
+
+  h3 {
+    margin-bottom: 0.5rem;
+  }
+
+  ul{
+    margin: 0;
+  }
+
+  li {
+    line-height: 0.75rem;
+  }
+  
+  p {
+    line-height: 1rem;
   }
 }
+
```

Comment:
Media queries, when we get around 861px, the container of the input and the button are bigger than the wrapper,  which bleeds out and makes the user  escrow to the side